hosts_example.yml

```diff
@@ -178,12 +178,44 @@ schema_registry:
       ## For kerberos sasl protocol, EACH host will need these two variables:
       # schema_registry_kerberos_keytab_path: <The path on ansible host to keytab file, eg. /tmp/keytabs/schemaregistry-ip-172-31-34-246.us-east-2.compute.internal
       # schema_registry_kerberos_principal: The principal configured in kdc server ex: schemaregistry/ip-172-31-34-246.us-east-2.compute.internal@REALM.EXAMPLE.COM>
+
+
 kafka_connect:
-  hosts:
-    ip-172-31-34-246.us-east-2.compute.internal:
+  children:
+    syslog1:
+      vars:
+        kafka_connect_group_id: connect-syslog1
```

Comment:
whats the best way of documenting this...?

Reply:
I added some similar documentation for rack.id into hosts.rack in the following PR.
https://github.com/domenicbove/cp-ansible/pull/7